packages/apps/tools/src/components/Global/ModuleExplorer/SidePanel/styles.css.ts

```diff
@@ -0,0 +1,14 @@
+import { style } from '@vanilla-extract/css';
+
+export const containerStyle = style([
+  { display: 'flex', flexDirection: 'column', height: '85vh' },
```

Comment:
We can use sprinkles here for all the values that are already defined

Reply:
Will take a look, thanks!

Reply:
I've refactored it to use values coming from `vars`. The UI/UX still needs to be done for this page, so the values used will most definitely change in the near future.